src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -331,7 +331,7 @@ public static Delegate Remove(Delegate source, Delegate value)
                 return source;
 
             if (!InternalEqualTypes(source, value))
-                throw new ArgumentException("The types of 'source' and 'value' should match.");
+                throw new ArgumentException($"The types of '{nameof(source)}' and '{nameof(value)}' should match.");
```

Comment:
This is localization bug - could you please add a proper `Arg_DlgtTypeMis` resource string for this instead, just like it is in CoreCLR?